### PR TITLE
Removing pushd/popd

### PR DESCRIPTION
**Pull request by silverdaz**
Andreas recommended to not use `pushd`/`popd`

### CODE REVIEW

File: docker/bootstrap/cega.sh [hunk 1]
```diff
@@ -107,15 +107,17 @@ EOF
...
+    ln -s ../john.yml .
+    ln -s ../jane.yml .
+    ln -s ../taylor.yml .
+)
 # John has also access to FIN1
-pushd $ABS_PRIVATE/cega/users/fin1 > /dev/null
-ln -s ../john.yml .
-popd > /dev/null
+(
+    cd $ABS_PRIVATE/cega/users/fin1 > /dev/null
```
Comment (viklund) [inline docker/bootstrap/cega.sh]: Do you really need the stdout redirect here?

Comment (silverdaz) [reply docker/bootstrap/cega.sh]: Oops. A miss.

**Comment by silverdaz** [comment]
I just wanted to rewrite the history, and squash the miss commit into the previous one.
How should I then push?

**Comment by viklund** [comment]
You can't not force push if you want to rewrite history that is already pushed. The only way to not force push is to have 2 commits. I don't think it's a big deal in this issue.

**Pull request merged**